### ISSUE TITLE
Scroller Pane value doesnt update when switching spectrum

### ISSUE THREAD

**Issue by BirbIrl** [tweaked-programs/cccbridge#54]
There is currently a bug with the slider that when enabling the negative spectrum for the scroll panel, you can't go from +X to -X without going to +Y then to -X. I.e: From 5 you can't go to -5, but you can go from 5 to -4, -6, -7 etc.

This is due to the fact that the slider seems to get confused where it really is. When selecting a positive value when the negative values are enabled, the slider after locking in the positive value, still loads in the negative slider's position every time you try and open the menu - instead of the positive slider right under.

1.19.2
1.5
Fabric

**Comment by BirbIrl** [comment]
also, when you set the limit to a negative number it gets REALLY messed up, is the best way i can describe it. Just lock the panel if the limit is a negative number.
Also also when the number is really big (idk i just threw in a 9999999) it crashed my client when right clicking the ui so there should probably be a lock for that too-

**Comment by SammyForReal** [comment]
If I understood you right, then the described bugs inside the UI seems to be something outside of our powers of fixing this. This feature is from Create, so you might want to open this in the [Fabricators-of-Create repository](https://github.com/Fabricators-of-Create/Create/) _(Correct me if I misunderstood you)_

Other than that, the desribed crashes or REALLY messed up behaviour seems to be something we didnt had in mind.
Thank you for taking the time with this, I will try to fix this quickly!

**Comment by BirbIrl** [comment]
Yeah the negative number issue seems more of a create issue! I wasn't sure if the mod can interface with how negative numbers work.

**Comment by SammyForReal** [comment]
Oh the bug with switching the direction (aka minus spectrum) seems to be a bug on our side, sorry!

**Comment by SammyForReal** [comment]
Alright, that setLimit crash got fixed now. That thing with not applying when switching minus and non-minus spectrum however not. I will keep this in mind however _(and keep this one open because of this)_

**Comment by SammyForReal** [comment]
Closing this since that issue is from a long time ago. If this issue is within the CC:C codebase, feel free to reopen and point at the code then please.
